apps/projects/app/components/Content/Overview.js

```diff
@@ -2,73 +2,42 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
 
-import { STATUS } from '../../utils/github'
-import { Project, Empty, Error } from '../Card'
-import Unauthorized from './Unauthorized'
-import { LoadingAnimation } from '../Shared'
-import { EmptyWrapper } from '../Shared'
-import { Viewport } from '@aragon/ui'
+import { Project, Empty } from '../Card'
+import { useLayout } from '@aragon/ui'
 import { CARD_STRETCH_BREAKPOINT } from '../../utils/responsive'
 
-const Overview = ({
-  changeActiveIndex,
-  onLogin,
-  projects,
-  githubLoading,
-  status,
-}) => {
-  if (githubLoading) {
-    return (
-      <EmptyWrapper>
-        <LoadingAnimation />
-      </EmptyWrapper>
-    )
-  } else if (status === STATUS.INITIAL) {
-    return <Unauthorized onLogin={onLogin} />
-  } else if (status === STATUS.FAILED) {
-    return <Error action={() => {}} />
-  }
+const Overview = ({ changeActiveIndex, projects }) => {
+  const { width } = useLayout()
 
   const projectsEmpty = projects.length === 0
   if (projectsEmpty) {
     return <Empty />
   }
 
   return (
-    <Viewport>
-      {({ width }) => (
-        <StyledProjects screenSize={width}>
-          {projects.map((project, index) => (
-            <Project
-              key={index}
-              label={project.metadata.name}
-              description={project.metadata.description}
-              id={project.id}
-              repoId={project.data._repo}
-              commits={project.metadata.commits}
-              screenSize={width}
-              // TODO: Disabled for now
-              // contributors={project.metadata.collaborators}
-              url={project.metadata.url}
-              changeActiveIndex={changeActiveIndex}
-            />
-          ))}
-        </StyledProjects>
-      )}
-    </Viewport>
+    <StyledProjects screenSize={width}>
```

Comment:
When the `screenSize`props changes the whole `projects.map` will be recomputed. A good optimization should be to perform the mapping in a function outside the return or in another component, and even `memoize` it so we avoid unneeded re-renders